test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -391,40 +399,40 @@ std::string RandomCode::fillArguments(eth::Instruction _opcode, RandomCodeOption
 			return code;
 		case eth::Instruction::CREATE:
 			//(CREATE value mem1 mem2)
-			code += getPushCode(randOpMemrGen());	//memlen1
-			code += getPushCode(randOpMemrGen());	//memlen1
+			code += getPushCode(randoOpSmallMemrGen());	//memlen1
+			code += getPushCode(randoOpSmallMemrGen());	//memlen1
 			code += getPushCode(randUniIntGen());	//value
 			return code;
 		case eth::Instruction::CALL:
 		case eth::Instruction::CALLCODE:
 			//(CALL gaslimit address value memstart1 memlen1 memstart2 memlen2)
 			//(CALLCODE gaslimit address value memstart1 memlen1 memstart2 memlen2)
-			code += getPushCode(randOpMemrGen());	//memlen2
-			code += getPushCode(randOpMemrGen());	//memstart2
-			code += getPushCode(randOpMemrGen());	//memlen1
-			code += getPushCode(randOpMemrGen());	//memstart1
+			code += getPushCode(randoOpSmallMemrGen());	//memlen2
+			code += getPushCode(randoOpSmallMemrGen());	//memstart2
+			code += getPushCode(randoOpSmallMemrGen());	//memlen1
+			code += getPushCode(randoOpSmallMemrGen());	//memstart1
 			code += getPushCode(randUniIntGen());	//value
-			code += getPushCode(toString(_options.getRandomAddress()));//address
+			code += getPushCode(toString(_options.getRandomAddress(RandomCodeOptions::AddressType::CallAccount)));//address
 			code += getPushCode(randUniIntGen());	//gaslimit
 			return code;
 		case eth::Instruction::STATICCALL:
 		case eth::Instruction::DELEGATECALL:
 			//(CALL gaslimit address value memstart1 memlen1 memstart2 memlen2)
 			//(CALLCODE gaslimit address value memstart1 memlen1 memstart2 memlen2)
-			code += getPushCode(randOpMemrGen());	//memlen2
-			code += getPushCode(randOpMemrGen());	//memstart2
-			code += getPushCode(randOpMemrGen());	//memlen1
-			code += getPushCode(randOpMemrGen());	//memstart1
-			code += getPushCode(toString(_options.getRandomAddress()));//address
+			code += getPushCode(randoOpSmallMemrGen());	//memlen2
+			code += getPushCode(randoOpSmallMemrGen());	//memstart2
+			code += getPushCode(randoOpSmallMemrGen());	//memlen1
+			code += getPushCode(randoOpSmallMemrGen());	//memstart1
+			code += getPushCode(toString(_options.getRandomAddress(RandomCodeOptions::AddressType::CallAccount)));//address
 			code += getPushCode(randUniIntGen());	//gaslimit
 			return code;
 		case eth::Instruction::SUICIDE: //(SUICIDE address)
 			code += getPushCode(toString(_options.getRandomAddress()));
 			return code;
 		case eth::Instruction::RETURN:  //(RETURN memlen1 memlen2)
 		case eth::Instruction::REVERT:  //(REVERT memlen1 memlen2)
-			code += getPushCode(randOpMemrGen());	//memlen1
-			code += getPushCode(randOpMemrGen());	//memlen1
+			code += getPushCode(randoOpSmallMemrGen());	//memlen1
```

Comment:
would it be better to change randOpMemrGen rather then creating a new dist?
is randOpMemrGen used anywhere now?